src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,165 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import PinNumberIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import Image from "next/image";
+import { UserBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { usePathname } from "next/navigation";
+import { Button } from "../../../../../../../components/server/Button";
+import Link from "next/link";
+import { useState } from "react";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "PIN";
+  breachData?: UserBreaches; //TODO: Remove conditional when other high-risk data categories are available on the BE
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  // Using mocked data for now, except for SSN breaches
+  const mockedData = [
+    { Name: "Nike", AddedDate: new Date("2013-12-07T14:48:00.000Z") },
+    { Name: "Adidas", AddedDate: new Date("2013-12-07T14:48:00.000Z") },
+    { Name: "Puma", AddedDate: new Date("2013-12-07T14:48:00.000Z") },
+  ];
+
+  const [isBreachResolved, setIsBreachResolved] = useState(false);
+  //TODO: Build out other high-risk data breach types - Remaining credit card, pin, bank account.
+  const exposedData = (() => {
+    if (props.breachData && props.typeOfBreach === "ssnBreaches")
+      return props.breachData.ssnBreaches;
+    if (props.typeOfBreach === "creditCard") return mockedData;
+    if (props.typeOfBreach === "bankAccount") return mockedData;
+    if (props.typeOfBreach === "PIN") return mockedData;
+    return [];
+  })();
+
+  // TODO: Make locale location-sensitive in the future
+  const dateFormatter = new Intl.DateTimeFormat("en-US", {
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle
+    dateStyle: "short",
+  });
+
+  const listOfBreaches = exposedData
+    .map((item) => `${item.Name} on ${dateFormatter.format(item.AddedDate)}`)
+    .join(", ");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat

Although we'll still need to localize that "on" in the above sentence.